app/views/world_wide_taxons/accordion.html.erb

```diff
@@ -38,7 +48,24 @@
             }
           %>
         <% end %>
-        <%= render 'govuk_publishing_components/components/accordion', items: items %>
+        <div data-module="gem-track-click gtm-click-tracking">
```

Comment:
`gtm-click-tracking` can be passed as a `module` option in the `data_attributes` passed to the accordion component (instead of needing to wrap the component in an extra div), like so:

```
<%= render 'govuk_publishing_components/components/accordion', {
  data_attributes: {
    module: "gtm-click-tracking",
  },
  data_attributes_show_all: {
    "ga4": show_all_attributes.to_json
  },
  items: items
} %>
```